openc3/lib/openc3/io/json_api.rb

```diff
@@ -53,7 +53,7 @@ def _generate_url(microservice_name:, prefix:, schema: 'http', hostname: nil, po
     # generate the auth object
     def _generate_auth
       if ENV['OPENC3_API_TOKEN'].nil? and ENV['OPENC3_API_USER'].nil?
-        if ENV['OPENC3_API_PASSWORD'] || ENV['OPENC3_SERVICE_PASSWORD']
+        if ENV['OPENC3_API_PASSWORD']
```

Comment:
`ENV['OPENC3_SERVICE_PASSWORD']` was never used right?

Reply:
Not really... at least it shouldn't have been.